Production/govuk_ios/Helpers/DateHelper.swift

```diff
@@ -0,0 +1,51 @@
+import Foundation
+
+class DateHelper {
+    static func getDay(date: Date) -> Int? {
+        let component = returnCalanderComponent(date: date)
+        guard let day = component.day else { return nil }
+        return day
+    }
+
+    static func getMonthName(components: DateComponents) -> String {
+        guard let componentMonth = components.month else { return "" }
+        return  DateFormatter().monthSymbols[componentMonth - 1]
+    }
+
+    static func getMonthAndYear(date: Date) -> String {
+        let components = returnCalanderComponent(date: date)
+        let monthName = getMonthName(components: components)
+        let year = components.year
+        guard let year = year else { return ""}
+        return "\(monthName) \(year)"
+    }
+
+    static func checkDatesAreTheSame(dateOne: Date, dateTwo: Date) -> Bool {
```

Comment:
This name is a little misleading, its isn't checking if the dates are the same, just that they are the same day.